pkg/storage/tsdb/block/index_test.go

```diff
@@ -59,9 +59,18 @@ func TestRewrite(t *testing.T) {
 
 	defer cw.Close()
 
+	tocalChunks := 0
+	ignoredChunks := 0
 	require.NoError(t, rewrite(log.NewNopLogger(), ir, cr, iw, cw, m, []ignoreFnType{func(mint, maxt int64, prev *chunks.Meta, curr *chunks.Meta) (bool, error) {
-		return curr.MaxTime == 696, nil
+		tocalChunks++
+		if curr.MinTime <= 600 && 600 <= curr.MaxTime {
```

Comment:
you can use `curr.OverlapsClosedInterval(600, 600)`

Reply:
also maybe a comment that you're arbitrarily ignoring just one chunk would help. Or that `600` is arbitrary